### DISCUSSION TITLE
PocketBase Data & Auth & Live Provider

### DISCUSSION THREAD

**Discussion by omeraplak** [refinedev/refine#3468] [Integration request] [answered]
### Is your feature request related to a problem? Please describe.

[PocketBase](https://pocketbase.io/) is an Open Source backend service. Writing [Data](https://refine.dev/docs/api-reference/core/providers/data-provider/), [Auth](https://refine.dev/docs/api-reference/core/providers/auth-provider/), and [Live](https://refine.dev/docs/api-reference/core/providers/live-provider/) Provider for refine would be great!

### Additional context

It might help to review [Supabase](https://github.com/pankod/refine/tree/next/packages/supabase), [Appwrite](https://github.com/pankod/refine/tree/next/packages/appwrite) packages

### Describe the thing to improve

PocketBase has a growing community. We think it would be a great pairing with refine 🍕

**Comment by kumar-kalyan** [comment]
Hey there @omeraplak can I work  on this issue

**Comment by omeraplak** [comment]
I did it! Happy hacking 🚀

**Comment by kumar-kalyan** [comment]
Hey @omeraplak  to  solve this  issue  I have to create a package for integrating pocket base right ?

**Comment by omeraplak** [comment]
> Hey @omeraplak to solve this issue I have to create a package for integrating pocket base right ?

Hey @kumar-kalyan ,
Yes, you are right! 
It might help to review [Supabase](https://github.com/pankod/refine/tree/next/packages/supabase), [Appwrite](https://github.com/pankod/refine/tree/next/packages/appwrite) packages

**Comment by salihozdemir** [comment]
Hey @kumar-kalyan,  Is there any progress on this issue?

**Comment by aliemir** [comment]
Due to inactivity, unassigning @kumar-kalyan from this issue. If anyone interested in this, please let us know! 🚀

**Comment by kruschid** [comment, marked as answer]
I'm currently developing a provider for PocketBase, [here](https://github.com/kruschid/refine-pocketbase). Your insights and contributions to the project would be highly valued. Any help with the code or feedback is welcome. 🙏

**Reply by omeraplak** [reply]
Wow, thank you very much for this valuable contribution

**Reply by kruschid** [reply]
thank you for the stars 🙏
